bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -26,9 +26,8 @@ stderr_print() {
     local bool="${BITNAMI_QUIET:-false}"
     # comparison is performed without regard to the case of alphabetic characters
     shopt -s nocasematch
-    if ! [[ "$bool" = 1 || "$bool" =~ ^(yes|true)$ ]]; then
-        printf "%b\\n" "${*}" >&2
-    fi
+    ! [[ "$bool" = 1 || "$bool" =~ ^(yes|true)$ ]] && printf "%b\\n" "${*}" >&2
```

Comment:
It may be that we are more used to the previous conditional structure than this one, but please revert this change for all occurrences.